tsdb/engine/tsm1/float.go

```diff
@@ -180,15 +188,15 @@ func (it *FloatDecoder) Next() bool {
 		return false
 	}
 
-	if bit == bitstream.Zero {
+	if bit == false {
 		// it.val = it.val
 	} else {
```

Comment:
This would read better if the core logic was just in one block that was `if bit {` rather than the dance of comparing a boolean variable to `false` and then using the `else` block.